source/includes/api/tables/progress-response.rst

```diff
@@ -0,0 +1,62 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 20 14 66
+
+   * - Field
+     - Type
+     - Description
+
+   * - ``state``
+     - string
+     - The current state of {+c2c-product-name+}.
+
+   * - ``canCommit``
+     - boolean
+     - If ``true``, indicates that a :ref:`commit <c2c-api-commit>`
+       request will succeed. This also means that the initial sync has
+       completed and is applying change events.
+
+   * - ``info``
+     - string
+     - Provides extra information on the synchronization progress.
+       Possible ``info`` values include:
+
+       - ``"change event application"``
```

Comment:
let's also add "collection copy". This is one customers will commonly see

Reply:
Done.